{"uuid": "13a06f9c-7dd0-400a-b449-36ad89a23442", "children": ["faf52162-971b-4905-840c-24ca33e3752f"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741860121641, "stop": 1741860121641}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 105, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741860121857, "stop": 1741860121863}], "start": 1741860121641, "stop": 1741860121863}